packages/sthrift/service-token-validation/src/index.ts

```
@@ -72,7 +72,6 @@ export class ServiceTokenValidation implements ServiceBase<TokenValidation> {
 				}
 			} catch {
 				// Required error handling, logging omitted to prevent flooding logs
-				continue;
```

Did removing this continue affect the login of admin user? I added this try/catch yesterday but wasn't sure if it would work with an empty catch so just double check